stable/vpa/templates/webhooks/jobs/certgen-patch.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if and .Values.admissionController.enabled .Values.admissionController.generateCertificate }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name:  {{ include "vpa.fullname" . }}-admission-certgen-patch
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    app.kubernetes.io/component: admission-certgen
+    {{- include "vpa.labels" . | nindent 4 }}
+spec:
```

Comment:
We should add a `ttlSecondsAfterFinished` here and for other jobs. Maybe `300` (5 minutes)?

I generally prefer this to the helm-hook strategy, since hooks can cause so many problems (e.g. with GitOps strategies)